### PR TITLE
Update centralized-deployment-of-add-ins.md

### PR DESCRIPTION
**Pull request by ericspli**
Added Role Requirements and consistency to section Titles.

### CODE REVIEW

**Comment by ericspli** [comment]
@microsoft-github-policy-service agree company="Microsoft"

File: microsoft-365/admin/manage/centralized-deployment-of-add-ins.md [hunk 1]
```diff
@@ -73,13 +73,28 @@ Centralized Deployment doesn't support the following:
 
     *MSI versions of Outlook show admin-installed add-ins in the appropriate Outlook ribbon, not the "My add-ins" section.
 
-### Exchange Online requirements
+### Exchange Online Requirements
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/centralized-deployment-of-add-ins.md]:
```suggestion
### Exchange Online requirements
```

File: microsoft-365/admin/manage/centralized-deployment-of-add-ins.md [hunk 2]
```diff
@@ -73,13 +73,28 @@ Centralized Deployment doesn't support the following:
...
 
 Microsoft Exchange stores the add-in manifests within your organization's tenant. The admin deploying add-ins and the users receiving those add-ins must be on a version of Exchange Online that supports OAuth authentication.
 
 Check with your organization's Exchange admin to find out which configuration is in use. OAuth connectivity per user can be verified by using the [Test-OAuthConnectivity](/powershell/module/exchange/test-oauthconnectivity) PowerShell cmdlet.
 
-### Admin requirements
+### Exchange Online User Role Requirements
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/centralized-deployment-of-add-ins.md]:
```suggestion
### Exchange Online user role requirements
```

File: microsoft-365/admin/manage/centralized-deployment-of-add-ins.md [hunk 3]
```diff
@@ -73,13 +73,28 @@ Centralized Deployment doesn't support the following:
...
+
+Use the classic Exchange Admin Center (EAC) to assign permissions to users. The permissions required to view and modify deployed add-ins are below.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/centralized-deployment-of-add-ins.md]:
```suggestion
Use the classic Exchange admin center (EAC) to assign permissions to users. The permissions required to view and modify deployed add-ins are detailed in the following steps.
```

File: microsoft-365/admin/manage/centralized-deployment-of-add-ins.md [hunk 4]
```diff
@@ -73,13 +73,28 @@ Centralized Deployment doesn't support the following:
...
+
+1.	Log in to the classic EAC as a global administrator.
+2.	Go to **Permissions**, and then select **User Roles**.
+3.	Select an existing role assignment policy or create a new one.
+4.	Type a name for the policy if you are creating a new one.
+5.	Select the following roles: **My Custom Apps**, **My MarketPlace Apps**, and **My ReadWriteMailbox Apps**.
+6.	Click Save.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/centralized-deployment-of-add-ins.md]:
```suggestion
1.	Log in to the classic EAC as a global administrator.
2.	Go to **Permissions** and then select **User Roles**.
3.	Select an existing role assignment policy or create a new policy.
4.	Type a name for the policy if you are creating a new policy.
5.	Select the following roles: **My Custom Apps**, **My MarketPlace Apps**, and **My ReadWriteMailbox Apps**.
6.	Select **Save**.
```

File: microsoft-365/admin/manage/centralized-deployment-of-add-ins.md [hunk 5]
```diff
@@ -73,13 +73,28 @@ Centralized Deployment doesn't support the following:
...
+
+*Note: These roles are selected by default.*
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/centralized-deployment-of-add-ins.md]:
```suggestion
> [!NOTE]
> These roles are selected by default.
```

File: microsoft-365/admin/manage/centralized-deployment-of-add-ins.md [hunk 6]
```diff
@@ -73,13 +73,28 @@ Centralized Deployment doesn't support the following:
...
+1.	Log in to the classic EAC as a global administrator.
+2.	Go to **Permissions**, and then select **User Roles**.
+3.	Select an existing role assignment policy or create a new one.
+4.	Type a name for the policy if you are creating a new one.
+5.	Select the following roles: **My Custom Apps**, **My MarketPlace Apps**, and **My ReadWriteMailbox Apps**.
+6.	Click Save.
+
+*Note: These roles are selected by default.* 
+
+For detailed information about how to do this, see [Manage role groups in Exchange Online](https://learn.microsoft.com/exchange/permissions-exo/role-groups). For detailed description of the different roles, see [Role assignment policies in Exchange Online](https://learn.microsoft.com/exchange/permissions-exo/role-assignment-policies).
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/centralized-deployment-of-add-ins.md]:
```suggestion
For more information, see [Manage role groups in Exchange Online](/exchange/permissions-exo/role-groups). For a detailed description of the different roles, see [Role assignment policies in Exchange Online](/exchange/permissions-exo/role-assignment-policies).
```

File: microsoft-365/admin/manage/centralized-deployment-of-add-ins.md [hunk 7]
```diff
@@ -73,13 +73,28 @@ Centralized Deployment doesn't support the following:
...
+
+### Admin Requirements
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/centralized-deployment-of-add-ins.md]:
```suggestion
### Admin requirements
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @ericspli, kindly commit the suggested changes. Note that all headings should be sentence case. Many thanks.

**Comment by ericspli** [comment]
@JohanFreelancer9 , Done.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@efrene Please merge this PR. Thanks

**Pull request merged**